tests/algorithm.cpp

```diff
@@ -0,0 +1,49 @@
+#include <rsl/algorithm.hpp>
+
+#include <catch2/catch_test_macros.hpp>
+#include <vector>
+
+using namespace rsl;
+
```

Comment:
```suggestion
```
The whole point of using a good, short namespace is that it's not encouraged to omit the namespace. I really want us using the namespace in our tests so we know what it's like to deal with the namespaces we pick. If it sucks to write out our namespace or symbol names, that's good feedback.